containers/azure-terraform/devcontainer-template.json

```diff
@@ -0,0 +1,48 @@
+{
+	"id": "azure-terraform",
+	"version": "1.0.0",
+	"name": "Azure Terraform",
+	"description": "_build, change, and version Azure infrastructure with terraform_",
+	"documentationURL": "https://github.com/microsoft/vscode-dev-containers/tree/main/containers/azure-terraform",
+	"publisher": "Community",
+	"licenseURL": "https://github.com/microsoft/vscode-dev-containers/blob/main/LICENSE",
+	"options": {
+		"installZsh": {
```

Comment:
Given the base image has ZSH and package upgrades in it, we could probably drop the installZsh and upgradePackages options here.

We also have a terraform feature, so technically this definition could just flip to using that.